internal/commands/push/command.go

```diff
@@ -444,3 +444,40 @@ func deployDraftAndWait(ui terminal.UI, realmClient realm.Client, remote appRemo
 	ui.Print(terminal.NewTextLog("Deployment complete"))
 	return nil
 }
+
+func importDependenciesAndWait(ui terminal.UI, realmClient realm.Client, remote appRemote, uploadPath string) error {
+	err := realmClient.ImportDependencies(remote.GroupID, remote.AppID, uploadPath)
+	if err != nil {
+		return err
+	}
+
+	s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+	s.Suffix = " Installing dependencies..."
+
+	waitForInstallation := func() error {
+		s.Start()
+		defer s.Stop()
+
+		status := realm.DependenciesStatusCreated
+		for status == realm.DependenciesStatusCreated {
+			time.Sleep(time.Second)
+			install, err := realmClient.DependenciesInstallation(remote.GroupID, remote.AppID)
+			if err != nil {
+				return err
+			}
+			s.Suffix = fmt.Sprintf(" %s...", install.StatusMessage)
+			status = install.Status
+			if status == realm.DependenciesStatusFailed {
+				return fmt.Errorf("failed to install dependencies")
+			}
+		}
+		return nil
+	}
+
+	if err := waitForInstallation(); err != nil {
+		return err
+	}
+
+	ui.Print(terminal.NewTextLog("Dependencies installation complete"))
```

Comment:
is this a good message to end with?

Reply:
alternatively could just use `"Installed dependencies"` (it sort of matches the rough pattern with the other spinners we've used in for importing hosting assets, resetting the cdn cache, etc)